### PR TITLE
chore(arm): Doc changes and cleanup of modules in examples

### PR DESCRIPTION
**Pull request by Jossuecito**
Doc changes and cleanup of modules in examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,19 +20,41 @@ Usage:
...
 
 Flags:
-  -h, --help   help for arm
+  -h, --help              help for arm
+  -l, --loglevel string   log level (default "info")
 
 Use "arm [command] --help" for more information about a command.
 ```
 
+##Mac OS
```
Comment (dogonthehorizon) [inline README.md:32]: Need to include a space between `#` and `M` so the GH MD render doesn't break:
https://s.armory.io/jkulvrB2

File: README.md [hunk 2]
```diff
@@ -22,19 +20,41 @@ Usage:
...
+Since arm-cli is not signed you may receive a couple of messages regarding security. To execute the binaries you need to:
+
+1.- When you execute arm-cli on the console you may see this message.
```
Comment (dogonthehorizon) [inline README.md:35]: I think the dashes (`-`) break rendering too :( Any reason to keep them?

File: README.md (hunk 1 above)
Comment (Jossuecito) [reply README.md]: Changed! Thanks!

File: README.md (hunk 2 above)
Comment (Jossuecito) [reply README.md]: Changed them also!! Thanks!

**Pull request merged**